exercise_0/terminalMode/TerminalMode.cpp

```diff
@@ -0,0 +1,16 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include <iostream>
+#include "TerminalMode.h"
+
+
+void TerminalMode::run() {
+    CharacterCounter characterCounter;
+    while (cin >> inputText && inputText != exit){
```

Comment:
Please keep the user informed that is "exit" button (i.e. by additional print, just before collecting characters).